src/viam/sdk/registry/registry.hpp

```diff
@@ -32,10 +32,7 @@ class ResourceSubtype {
    public:
     virtual ~ResourceSubtype();
     /// @brief Add `Reconfigure` functionality to a resource.
-    std::function<Resource(Resource, Name)> create_reconfigurable;
-
-    // TODO: it doesn't look like we actually use this. Confirm, then remove.
-    std::function<ProtoType(Resource)> create_status;
```

Comment:
(flyby) had to touch this signature anyways, seemed an easy time to follow up on the `TODO`.